packages/components/package.json

```diff
@@ -82,6 +82,7 @@
     "eslint-import-resolver-webpack": "^0.11.1",
     "formik": "^2.1.4",
     "gh-pages": "^2.1.1",
+    "mobx-react": "^6.2.2",
```

Comment:
Is there another way to handle this so we don't need to add `mobx-react` or the `connect` function to components?

Reply:
This approach is more efficient as we only render the specific component which needs to re-rendered. 
Alternatively, we can make the parent component of `ContractCard` to re-render. But connecting to observables to cause re-render should be done separately in both `DTrader` and `DBot`.